release_dashboard/lib/widgets/engine_cherrypicks_substeps.dart

```diff
@@ -62,39 +76,89 @@ class ConductorSubstepsState extends State<EngineCherrypicksSubsteps> {
-          subtitle: SelectableText(EngineCherrypicksSubsteps.substepSubtitles[EngineCherrypicksSubstep.substep1]!),
-          isChecked: _isEachSubstepChecked[EngineCherrypicksSubstep.substep1]!,
+          substepName: EngineCherrypicksSubsteps.substepTitles[EngineCherrypicksSubstep.verifyRelease]!,
+          subtitle: Column(
+            crossAxisAlignment: CrossAxisAlignment.start,
+            children: [
+              SelectableText(
+                  'Verify if the release number: ${context.watch<StatusState>().releaseStatus?['Release Version']}'
+                  ' is correct based on existing published releases here: '),
+              const UrlButton(
+                textToDisplay: EngineCherrypicksSubsteps.releaseSDKURL,
+                urlOrUri: EngineCherrypicksSubsteps.releaseSDKURL,
+              ),
+            ],
+          ),
+          isChecked: _isEachSubstepChecked[EngineCherrypicksSubstep.verifyRelease]!,
           clickCallback: () {
-            substepPressed(EngineCherrypicksSubstep.substep1);
+            substepPressed(EngineCherrypicksSubstep.verifyRelease);
           },
         ),
         CheckboxAsSubstep(
-          substepName: EngineCherrypicksSubsteps.substepTitles[EngineCherrypicksSubstep.substep2]!,
-          subtitle: SelectableText(EngineCherrypicksSubsteps.substepSubtitles[EngineCherrypicksSubstep.substep2]!),
-          isChecked: _isEachSubstepChecked[EngineCherrypicksSubstep.substep2]!,
+          substepName: EngineCherrypicksSubsteps.substepTitles[EngineCherrypicksSubstep.applyCherrypicks]!,
+          subtitle: engineCherrypicksInConflict.isEmpty
+              ? const SelectableText('No cherrypick conflicts, just check this substep.')
+              : Column(
+                  crossAxisAlignment: CrossAxisAlignment.start,
+                  children: [
+                    const SelectableText(
+                        "You must manually apply the following engine cherrypicks that are in conflict "
+                        "by doing 'git cherrypick [hash]' with the following hashes: "),
+                    SelectableText('${engineCherrypicksInConflict.join('\n')}\n'),
+                    const SelectableText('to the engine checkout at the following location: '),
+                    UrlButton(
+                      textToDisplay:
+                          '${context.watch<StatusState>().conductor.rootDirectory.path}/flutter_conductor_checkouts/engine',
+                      urlOrUri:
+                          '${context.watch<StatusState>().conductor.rootDirectory.path}/flutter_conductor_checkouts/engine',
+                    ),
+                    const SelectableText('See more information at: '),
+                    const UrlButton(
+                      textToDisplay: EngineCherrypicksSubsteps.cherrypickHelpURL,
+                      urlOrUri: EngineCherrypicksSubsteps.cherrypickHelpURL,
```

Comment:
Will the conductor state update after the cherry picks have been manually resolved? Could we rely on that to validate this step?

Reply:
The conductor core does not update the status of the cherrypicks after the user has resolved them. I can make a UI component to make the user to update them.

Reply:
There might be a problem with this. The release dashboard can only update the status, and not the state file. The status will be syncing with the state file in this [PR](https://github.com/flutter/flutter/issues/93156). If the UI only updates the cherrypick states of the status and not the state file, then after a state sync, the cherrypick states update will be gone. 
In order to achieve this, I am thinking we will probably need a conductor core function to update the cherrypicks status in the sate file? @christopherfujino 
Another solution I was thinking is when the cherrypick status is updated by the user, we assign it a special status like `Resolved`. And then when the status syncs with the state file, we do a check to see if the cherrypicks are `Resolved`. If they are, we leave them as they are, if not, take the status from the status file.
Or we could leave the cherrypicks status as `pending with conflict` even after the user has resolved them.

Reply:
@CaseyHillers and I just talked about this. Since the conductor core doesn't support updating cherrypick states. We will leave states un-updated.

Reply:
> Another solution I was thinking is when the cherrypick status is updated by the user, we assign it a special status like `Resolved`. And then when the status syncs with the state file, we do a check to see if the cherrypicks' states are `Resolved`. If they are, we leave them as they are, if not, take the status from the state file.

I think it would be a bug waiting to happen to have two sources of truth for this data, and have special logic to sync them. I would recommend just creating a new API in the core to update the status of a cherrypick. This could be a method on the `NextContext`, like `updateCherryPickStatus()`